### DISCUSSION TITLE
On how many PCs can I use ...

### DISCUSSION THREAD

**Discussion by Temporary1963** [aclap-dev/video-downloadhelper#576] [License & Purchase requests]
### What's your issue?

... the extension having paid for 1 license? 

Thanks

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You can use it on all your devices: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Reply by Temporary1963** [reply]
thanks , excuse me for the question . I did't noticed the faq answer .
